### ISSUE TITLE
Simple code is crashing my app

### ISSUE THREAD

**Issue by matimoto87** [PhilJay/MPAndroidChart#807]
Dear all!

I'm desperate. For three days I'm trying to make a simple linear chart in my app. Trying to do it in Android Studio 1.2.1.1 with API 22.

Let's start from XML file:

```xml

[...]

<FrameLayout
                android:layout_width="0dip"
                android:layout_height="match_parent"
                android:layout_weight="1"
                android:background="#008272"
                android:layout_margin="5dp" >

                <com.github.mikephil.charting.charts.LineChart
                    android:id="@+id/linechart"
                    android:layout_width="match_parent"
                    android:layout_height="match_parent" />


            </FrameLayout>

```

Now, MainActivity.java (onCreate method)

```javascript

protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        mNavigationDrawerFragment = (NavigationDrawerFragment)
                getSupportFragmentManager().findFragmentById(R.id.navigation_drawer);
        mTitle = getTitle();

        // Set up the drawer.
        mNavigationDrawerFragment.setUp(
                R.id.navigation_drawer,
                (DrawerLayout) findViewById(R.id.drawer_layout));

        LineChart lchart = (LineChart) findViewById(R.id.linechart);
        String text = String.valueOf(lchart.getId());
        Toast.makeText(getApplicationContext(), text, Toast.LENGTH_LONG).show();
    }
```

Even with this simple code, my app terminates all the time. 

First, Ive used a simple tutorial to make a chart. The attempt resulted in a crash. Today, the app doesn't work with above code.

Logcat:

```java

java.lang.RuntimeException: Unable to start activity ComponentInfo{com.mkilinski.spalanie/com.mkilinski.spalanie.MainActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'int com.github.mikephil.charting.charts.LineChart.getId()' on a null object reference

[...]

Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'int com.github.mikephil.charting.charts.LineChart.getId()' on a null object reference
            at com.mkilinski.spalanie.MainActivity.onCreate(MainActivity.java:70)

```

Line 70 : String text = String.valueOf(lchart.getId());

Always the same issue : null object reference.
I've make a question on [Stackoverflow][1] with more-detailed source code. 

Can someone help me? 

Greetings!

[1]:https://stackoverflow.com/questions/31045720/simple-mpandroidchart-use-gives-app-crash

**Comment by NeoRaidenX** [comment]
I read somewhere in the documentation that you have to specify the height in your xml LineChart and not just put "match_parent" or "wrap_content". Try changing the value and run it again. Hope this works.

Also, Why do you want to get the chart Id? For some reason lchart.getId() is sending to you a null value. But is really necessary to get the chart id?

**Comment by matimoto87** [comment]
Thanks, I will try. Called chartId is some kind of debugger. Because I noticed that any call to MCAndroidChart object is ending with crash.

**Comment by PhilJay** [comment]
You are getting a nullpointer exception, so obviously, your `lchart` is null.
You need to figure out why that happens (e.g. wrong .xml file)

**Comment by matimoto87** [comment]
Thanks. Strange situation - I've made a new project in Android Studio and then included lib by build.gradle file as **compile 'com.github.PhilJay:MPAndroidChart:v2.0.9'** entry and now everything works fine o_O

Earlier, I've included lib manually by copying jar file to lib folder and setting params in Project Structure. The second change was a version of library. First, I've used a 2.1.0 version, now I'm using the 2.0.9 version. 

That was a cause of my error, I think...

Anyway, thanks a lot!
